db/migrations/20220330120000_org_id_support.go

```diff
@@ -0,0 +1,131 @@
+package migrations
+
+import (
+	"context"
+	_ "embed"
+
+	"github.com/RedHatInsights/sources-api-go/config"
+	logging "github.com/RedHatInsights/sources-api-go/logger"
+	"github.com/RedHatInsights/tenant-utils/pkg/tenantid"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// OrgIdSupport creates a new "org_id" column with an index in the "tenants" table and attempts to translate the
+// current EBS account numbers to "org_id"s.
+//
+// If the translation is not possible, it doesn't fail the migration: it simply logs an error. This is due to the
+// transition stage we are in. Since we are proactively taking steps towards the full "OrgId" support, not having a
+// translation for an EBS account number is acceptable.
+//
+// It also adds a comment to the "external_tenant" column, clarifying that it refers to "EBS account numbers".
+func OrgIdSupport() *gormigrate.Migration {
```

Comment:
(1)I would prefer whether this name would describe more what it is doing(+name file).
"AddOrgIDtoTenants"
"Translate EBS account number to ORG ID"

(2) Also maybe it would be good to don't mix structural and data migrations.

Let me know what do you think about it.

Reply:
> (1)I would prefer whether this name would describe more what it is doing(+name file). "AddOrgIDtoTenants" "Translate EBS account number to ORG ID"

Sure. I am very bad at naming functions so that's why I end up writing comments on them to give them more context.

> (2) Also maybe it would be good to don't mix structural and data migrations.

Why is this? In this case the migration doesn't fail if the data migration does not succeed, because it's a "since we're at it, let's try populating the column too" kind of thing. That's why I conceived it as part of the same migration.

Reply:
(2) It is just my opinion - I think there is too much things in one migration:
- add column + add commenT
- translation 

it could be two migrations - reason is just to be more clear what is happening in one migration. I see one migration as one unit where less things (reasonable) as possible - and then then we don't need go to start back when anythings fail.
Also naming of migrations is more unambiguous without hiding additions in migration - you can see on first look what is going inside migration.
+ when we would test data migration it would be also more clear what is being tested.

yes and now I understand more role of the translation operation.

I don't persist on it -  I leave it  up to you - I wanted to see rather your opinion - If you like we can adopt it as "rule" migration.

I would like also hear @lindgrenj6's opinion

Reply:
> it could be two migrations - reason is just to be more clear what is happening in one migration. I see one migration as one unit where less things (reasonable) as possible - and then then we don't need go to start back when anythings fail.
> Also naming of migrations is more unambiguous without hiding additions in migration - you can see on first look what is going inside migration.

Okay that's a good point. After refactoring it is true that it is easier to follow. I submitted the commit.